Tests/SwiftProtobufTests/Test_TextFormat_proto3.swift

```diff
@@ -1389,4 +1396,87 @@ class Test_TextFormat_proto3: XCTestCase, PBTestHelpers {
 
         assertTextFormatEncode(expected, configure: configureLargeObject)
     }
+
+    func testMultipleFields_numbers() {
+        let text: String = ("1: 1\n"
+            + "2: 2\n"
+            + "3: 3\n"
+            + "4: 4\n"
+            + "5: 5\n"
+            + "6: 6\n"
+            + "7: 7\n"
+            + "8: 8\n"
+            + "9: 9\n"
+            + "10: 10\n"
+            + "11: 11\n"
+            + "12: 12\n"
+            + "13: true\n"
+            + "14: \"abc\"\n"
+            + "15: \"AB\"\n"
+            + "18 {\n"
+            + "  bb: 7\n"
+            + "}\n"
+            + "19 {\n"
+            + "  c: 88\n"
+            + "}\n"
+            + "20 {\n"
+            + "  d: -9\n"
+            + "}\n"
+            + "21: BAZ\n"
+            + "22: FOREIGN_BAZ\n"
+            + "23: IMPORT_BAZ\n"
```

Comment:
Can you spin an issue for that, I had assumed that was already support, and it seems independent of supporting using field numbers for input.